packages/eas-cli/src/graphql/types/credentials/IosAppBuildCredentials.ts

```diff
@@ -36,17 +39,5 @@ export const IosAppBuildCredentialsFragmentNode = gql`
         appleTeamName
       }
     }
-    appleDevices {
```

Comment:
i removed this because i deprecated it on the server (https://github.com/expo/universe/blob/master/server/www/src/graphql/types/credentials/IosAppBuildCredentials.ts#L27). We don't actually resolve `appleDevices` any more